x-pack/plugin/esql/qa/server/single-node/build.gradle

```diff
@@ -21,6 +21,10 @@ restResources {
 tasks.named('javaRestTest') {
   usesDefaultDistribution()
   maxParallelForks = 1
+  dependencies {
```

Comment:
We don't want this block inside the `tasks.named` block. Dependencies should be declared in the root of the build script.

Reply:
Done. Thanks @mark-vieira